src/common/mutations/createProfile.ts

```diff
@@ -4,7 +4,7 @@ export const CREATE_NEW_PROFILE_MUTATION = gql`
   mutation CREATE_NEW_PROFILE(
     $firstName: String!
     $lastName: String!
-    $addresses: [CreateAddressInput]
+    $addresses: [CreateAddressInput]!
```

Comment:
```suggestion
    $addresses: [CreateAddressInput!]!
```

So the generated type would resolve to

```
addresses: CreateAddressInput[]
```

instead of 

```
addresses: (CreateAddressInput | null)[];
```

my bad for originally messing this up.

Reply:
Sure, added a commit